_posts/2020-09-15-data-sharing-at-scale.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: post
+title: Data sharing at scale
+date: 2020-09-15
+tags: aws cloudformation lakeformation glue data
+image: img/../.jpg
+author: Zainab Maleki, Simon Bracegirdle
+---
+<center><img src="/img/lake-formation/binary-arrows_Web.jpg" /></center><br/>
+
+## Introduction
+
+If you're building a data platform for your organisation, you may be wondering how to make data in one business area accessible to other areas. You want the technology to support the way your organisation works and enable you to scale without creating headaches.
+
+For example, let’s say you’re building a dashboard as part of a solution for a customer. The dashboard aggregates data from stores belonging to other departments.
+
+<center><img src="/img/lake-formation/00.png" /></center><br/>
+
+How can we solve this problem for a data platform built on AWS? These data stores are in completely different AWS accounts belonging to different teams. How do we know what data is available in each of those stores? What is the schema of the data? How can we get access to it?
+
+If these sound like problems that are relevant to you, then please keep reading! We'll go through the steps of building a cross-account resource sharing solution with AWS Lake Formation.
+
+## Which AWS services can help?
+
+One service that can help is [AWS Glue](https://aws.amazon.com/glue), a managed service that enables crawling of data repositories to help assemble a data catalog. Glue also provides tools for Extract, Transform and Load (ETL), called Jobs.
```

Comment:
as was pointed out `catalog` is the american english - catalogue is british english (and by extension, australian) no big deal though

Reply:
I looked at AWS docs and they refer to it as AWS Glue Data Catalog. That's why I kept it as the American version